librarian.php

```diff
@@ -263,6 +260,29 @@ function count_feeds()
         $filecount = count(glob($g_dir_feeds . "/*.xml"));
         return $filecount;
     }
+
+    $param_url = fetch_param("url");
+    $param_id = fetch_param("id");
+    $param_delete = fetch_param("delete");
+    $user_id = $param_id;
+
+    if ($param_id != "" && $param_url != "")
+    {
+        if ($param_delete == "1")
+        {
+            $result = remove_url($param_id, $param_url);
+            http_response_code(302);
+            header('Location: /?id='.$param_id);
```

Comment:
Please change to `header('Location: ' .$g_url_librarian. '/?id='.$param_id);`

Reply:
This way the correct librarian.php file is called, instead of assuming the URL is at the root.